src/lib.rs

```diff
@@ -203,23 +203,58 @@ pub struct Metrics {
     pub strikeout_thickness: f32,
 }
 
-pub trait Rasterize {
-    /// Errors occurring in Rasterize methods.
-    type Err: ::std::error::Error + Send + Sync + 'static;
+/// Errors occuring when using the rasterizer.
+#[derive(Debug)]
+pub enum Error {
+    /// Couldn't find font matching description.
+    MissingFont(FontDesc),
+
+    /// Tried to get size metrics from a Face that didn't have a size.
+    MissingSizeMetrics,
+
+    MissingGlyph(RasterizedGlyph),
+
+    /// Requested an operation with a FontKey that isn't known to the rasterizer.
+    FontNotLoaded,
 
+    /// Error from platfrom's font system.
+    PlatformError(String),
+}
+
+impl std::error::Error for Error {
+    fn source(&self) -> Option<&(dyn std::error::Error + 'static)> {
+        None
+    }
+}
+
+impl Display for Error {
+    fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result {
+        match self {
+            Error::MissingFont(font) => write!(f, "Unable to find the font {}", font),
```

Comment:
```suggestion
            Error::MissingFont(font) => write!(f, "Unable to find the font {:?}", font),
```

To quote it, might make it easier to understand? Could solve potential issues like searching for `Fira Mono<lots of whitespace>`.